modules/monitoring/application/forms/Command/Object/ScheduleHostDowntimeCommandForm.php

```diff
@@ -15,6 +16,31 @@
  */
 class ScheduleHostDowntimeCommandForm extends ScheduleServiceDowntimeCommandForm
 {
+    /**
+     * @var bool
+     */
+    protected $hostDowntimeAllServices;
+
+    public function init()
+    {
+        $this->start = new DateTime();
+        $config = Config::module('monitoring');
+        $this->commentText =  $config->get('settings', 'comment_text', 'Reboot');
```

Comment:
Not sure if we should provide a default comment text out of the box. We didn't for years now. It's also not possible to disable it somehow, other than configuring an empty string.

If one doesn't define a default comment explicitly, there should be none. Please remove the default.

Reply:
Oh and the option should be `hostdowntime_comment_text`, there may be different comments needed for hosts and services. (Services can't be rebooted usually :laughing: )